src/checkstyle/checks.xml

```diff
@@ -35,6 +35,10 @@
         <property name="format" value="\n\n\Z" />
         <property name="message" value="Blank line before end of file" />
     </module>
+    <module name="RegexpMultiline">
+        <property name="format" value="->\s*\{\s+\}" />
+        <property name="message" value="Whitespace inside empty lambda body" />
```

Comment:
It does not necessarily mean that it is lambda.

Reply:
Ah there is `->`, then ok. 

What about `\s*` is possible to have there multiple or none space between `->` and `{}`? Maybe you can replace that with simple ` ` (single space)?

Reply:
Multiple (or no) spaces between `->` and `{` don't make spaces inside
`{..}` more legal.
I didn't want to unnecessarily depend on other rules here